### PR TITLE
[14.0][MIG] account_invoice_supplier_ref_unique: Migration to 14.0

### PR DESCRIPTION
**Pull request by miguel-S73**
Standard migration to 14.0 and adjust test.

### CODE REVIEW

File: account_invoice_supplier_ref_unique/views/account_move.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="view_move_form" model="ir.ui.view">
+        <field name="model">account.move</field>
+        <field name="inherit_id" ref="account.view_move_form" />
+        <field name="arch" type="xml">
+            <field name="ref" position="before">
```
Comment (chusamo) [inline account_invoice_supplier_ref_unique/views/account_move.xml]: Now

`<xpath expr="//label[@for='ref']" position="before">`


Captura de pantalla de 2021-03-20 08-35-31


Could you change please?

**Review by chusamo — APPROVED** [review]
Functional review LGTM

Could you check the view design please?

File: account_invoice_supplier_ref_unique/views/account_move.xml (hunk 1 above)
Comment (ferran-S73) [reply account_invoice_supplier_ref_unique/views/account_move.xml]: Done

**Comment by ferran-S73** [comment]
Not sure why the account_billing test is failing now. Any ideas?

**Comment by ps-tubtim** [comment]
@miguels73 Please try to rebase to fix Travis.

File: account_invoice_supplier_ref_unique/readme/CONTRIBUTORS.rst [hunk 2]
```diff
@@ -0,0 +1,5 @@
+* Marc Cassuto <marc.cassuto@savoirfairelinux.com>
+* Mathieu Benoit <mathieu.benoit@savoirfairelinux.com>
+* Alexis de Lattre <alexis.delattre@akretion.com>
+* Thomas Binsfeld <thomas.binsfeld@acsone.eu>
+* Andrea Stirpe <a.stirpe@onestein.nl>
```
Comment (ps-tubtim) [inline account_invoice_supplier_ref_unique/readme/CONTRIBUTORS.rst:5]: @miguels73 Please add you as a contributor on `CONTRIBUTORS.rst`.

Comment (miguel-S73) [reply account_invoice_supplier_ref_unique/readme/CONTRIBUTORS.rst:5]: Added

**Review by ps-tubtim — APPROVED** [review]

**Pull request merged**